.babelrc

```diff
@@ -3,11 +3,12 @@
     "react"
   ],
   "plugins": [
-    "transform-es2015-modules-amd",
     "check-es2015-constants",
     "transform-es2015-block-scoping",
     "transform-es2015-arrow-functions",
     "transform-es2015-parameters",
+    "transform-es2015-destructuring",
```

Comment:
That sounds good. Just enabling destructuring makes it work on Edge or is there anything else obvious to do?